src/plugins.js

```diff
@@ -271,7 +272,10 @@ export default class Plugins {
     const pjson = path.join(this.userPluginsDir, 'package.json')
     const yarnrc = path.join(this.userPluginsDir, '.yarnrc')
     fs.mkdirpSync(this.userPluginsDir)
-    if (!fs.existsSync(pjson)) fs.writeFileSync(pjson, JSON.stringify({private: true}))
```

Comment:
ah I didn't trace the code, might be best for me to fit it in as part of my plugins refactor then